ell-studio/src/components/source/DiffRenderer.js

```diff
@@ -12,6 +12,14 @@ export function DiffRenderer({
 }) {
   if (!previousCode) return null;
 
+  function removeEllTags(code) {
+    const ellTagRegex = /\s*#\s*<\/?(BV|BmV|LMP)>/g;
+    return code.replace(ellTagRegex, '');
+  }
+
+  code = removeEllTags(code);
+  previousCode = removeEllTags(previousCode);
```

Comment:
i think we dont want to remvoe tags. but if this is only for the diff renderer i guess thats fine ;)